engine/jni/scripts/external/gen_ir.py

```diff
@@ -208,16 +209,18 @@ def parse_inner_cpp(main_prefix, dep_prefixes, namespace, inp, outp):
 
 
 def clang(csrc_path, includes=[]):
-    cmd = ['clang', '-Xclang', '-ast-dump=json', '-c']
+    clang = os.environ.get('CLANG', 'clang')
+    cmd = [clang, '-Xclang', '-ast-dump=json', '-c']
```

Comment:
Now uses the clang we're found earlier.
Previously, it was required to be on PATH